carbon.yml

```diff
@@ -0,0 +1,707 @@
+  combo-box:
+    status: stable
+    framework: svelte
+    demoLinks:
+      - type: storybook
+        name: Storybook
+        action: link
+        url: https://carbon-components-svelte.onrender.com/components/ComboBox
+  composed-modal:
+    name: Composed modal
+    status: stable
+    type: component
+    platform: web
+    framework: svelte
+    thumbnailPath: './thumbnails/composed-modal.svg'
+    demoLinks:
+      - type: storybook
+        name: Storybook
+        action: link
+        url: https://carbon-components-svelte.onrender.com/components/ComposedModal
+    tags:
+      - input-control
+  content-switcher:
+    status: stable
+    externalDocsUrl: https://www.carbondesignsystem.com/components/content-switcher/usage
+    framework: svelte
+    demoLinks:
+      - type: storybook
+        name: Storybook
+        action: link
+        url: https://carbon-components-svelte.onrender.com/components/ContentSwitcher
+  context-menu:
+    name: Context menu
+    status: stable
+    type: component
+    platform: web
+    framework: svelte
+    thumbnailPath: './thumbnails/context-menu.svg'
+    demoLinks:
+      - type: storybook
+        name: Storybook
+        action: link
+        url: https://carbon-components-svelte.onrender.com/components/ContextMenu
+    #TODO: @aubrey to check tag
+    tags:
+      - input-control
+      - contextual-navigation
+  copy-button:
+    status: stable
+    framework: svelte
+    demoLinks:
+      - type: storybook
+        name: Storybook
+        action: link
+        url: https://carbon-components-svelte.onrender.com/components/CopyButton
+  data-table:
+    status: stable
+    framework: svelte
+    externalDocsUrl: https://www.carbondesignsystem.com/components/data-table/usage
+    demoLinks:
+      - type: storybook
+        name: Storybook
+        action: link
+        url: https://carbon-components-svelte.onrender.com/components/DataTable
+  date-picker:
+    status: stable
+    framework: svelte
+    externalDocsUrl: https://www.carbondesignsystem.com/components/date-picker/usage
+    demoLinks:
+      - type: storybook
+        name: Storybook
+        action: link
+        url: https://carbon-components-svelte.onrender.com/components/DatePicker
+  dropdown:
+    status: stable
+    framework: svelte
+    externalDocsUrl: https://www.carbondesignsystem.com/components/dropdown/usage
+    demoLinks:
+      - type: storybook
+        name: Storybook
+        action: link
+        url: https://carbon-components-svelte.onrender.com/components/Dropdown
+  expandable-tile:
+    name: Expandable tile
+    status: stable
+    type: component
+    platform: web
+    framework: svelte
+    thumbnailPath: './thumbnails/expandable-tile.svg'
+    demoLinks:
+      - type: storybook
+        name: Storybook
+        action: link
+        url: https://carbon-components-svelte.onrender.com/components/ExpandableTile
+    #TODO: @aubrey check tag
+    tags:
+      - input-control
```

Comment:
Delete input control. Add data-display and content element (has the same tags as Accordion).